src/app/ApplicationTemplate.Shared/Framework/Browser/DispatcherBrowserService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Core;
+using Xamarin.Essentials;
+using Xamarin.Essentials.Interfaces;
+
+namespace ApplicationTemplate.Framework
+{
+	/// <summary>
+	/// Invokes calls to an IBrowser on the dispatcher
+	/// </summary>
+	public class DispatcherBrowserService : IBrowser
```

Comment:
I feel like it should be named `DispatcherBrowserDecorator `since it's decorating an existing `IBrowser `with the dispatcher functionality.

Reply:
done